components/articles/EditArticleForm/EditLocaleForm.js

```diff
@@ -33,19 +34,17 @@ const localeFields = [
 // FIXME: it's very ugly & difficult method, has no ideas how to fix it
 // mb use NestedForm, but there are another bugs with it
 // TODO: migrate on react-form@3 & check NestedForms
-export const localesValidator = locales => {
-  const localesErrors = {};
-  const localesHasErrors = !!Object.keys(locales)
-    .map(loc => {
-      localeFields.forEach(({ field, validator }) => {
-        const path = `${loc}.${field}`;
-        set(localesErrors, path, validator(get(locales, path)));
-      });
-      return Object.values(localesErrors[loc]).some(Boolean);
-    })
-    .filter(Boolean).length;
-  return localesHasErrors ? localesErrors : null;
-};
+export const localesValidator = locales =>
+  Object.keys(locales).reduce((errors, loc) => {
+    localeFields.forEach(({ field, validator }) => {
+      const path = `${loc}.${field}`;
+      const validationError = validator(get(locales, path));
+
+      set(errors, path, validationError);
+    });
+
+    return errors;
+  }, {});
```

Comment:
in previous implementation `localesValidator` return `null` if there is no errors

now it returns `{}`

is't broke something?

Reply:
No. Now it returns object like `{
 be: {
  slug: null,
 }
}`